Dockerfile-cli

```diff
@@ -13,13 +13,14 @@ RUN set -x \
 COPY src/php/utils/docker/ /usr/local/bin/
 
 # Install PHP extensions
+# hadolint ignore=DL4006
 RUN set -x \
     # Adding sodium purely for the 7.1 image, it's already in 7.2 and up: https://www.php.net/manual/en/sodium.installation.php
-    && apk add --no-cache libsodium-dev wget \
+    && if [ $(php -v | grep "PHP 7.1" | wc -l) != 0 ] ; then apk add --no-cache libsodium-dev wget; else true; fi \
```

Comment:
:bulb: We could make the PHP version a build argument, so we don't need to evaluate it at build time.
We'd have something like this at the top of the Dockefile:

```
ARG PHP_VERSION=7.2
FROM php:${PHP_VERSION}-cli-alpine3.7 as cli
```

Reply:
We'd still have to do the if here, but yeah that would be a better solution. Once I figured out why this build is failing and how to fix it.

Reply:
Tbh I rather make this change in a different PR because the changes are a bit bigger than just the Dockerfiles.